features/support/local_server.rb

```diff
@@ -102,7 +102,7 @@ def generate_patch(subject)
 --- /dev/null
 +++ b/README.md
 +hello
--- 
+--
```

Comment:
Watch out when letting your editor clean up whitespace. This is inside a patch file, so it's best to avoid making any changes to it.